### DISCUSSION TITLE
How-to: Get the difference between two objects?

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [selfrefactor/rambda#669] [Q&A]
I think, making comparisons is a pretty common subject. So how can I use two objects to find how one differs from the other?

**Comment by selfrefactor** [comment]
this is not as straight-forward as it includes nested looping through all the properties. I use a library for this logic that has no relation to Rambda.

**Reply by sschneider-ihre-pvs** [reply]
I saw an example with lodash using recursion.

**Comment by sschneider-ihre-pvs** [comment]
https://gist.github.com/Yimiprod/7ee176597fef230d1451
this is one I used before.
